package.json

```diff
@@ -44,30 +44,32 @@
     "help": "nx help",
     "postinstall": "run-p --continue-on-error postinstall:*",
     "postinstall:deps": "exitzero electron-builder install-app-deps",
-    "postinstall:copy-themes": "shx cp node_modules/@elastic/eui/dist/*.min.css apps/postybirb-ui/src/assets",
+    "postinstall:copy-themes": "shx cp node_modules/@elastic/eui/dist/*.min.css apps/postybirb-ui/public",
+    "postinstall:patch-package": "patch-package",
     "setup": "run-s setup:husky",
-    "setup:husky": "husky install"
+    "setup:husky": "husky install",
+    "tools:icons": "ts-node tools/generateIcons.ts"
```

Comment:
question: thoughts about having postinstall call the generate icon tool?

Reply:
i dont think its needed, because icons list updates only when we change lib version